utils.sh

```diff
@@ -14,12 +14,63 @@ wd=$(mktemp -d)
 target_name="$(basename "$target")"
 
 current="$target"
-for patch in "${PWD}/ignition_patches/${kind}/"*.json; do
+for patch in $(ls "${PWD}/ignition_patches/${kind}/"*.json); do
     current_patch_name="$(basename "$patch")"
     jsonpatch "$current" "$patch" > "${wd}/${target_name}_${current_patch_name}"
     current="${wd}/${target_name}_${current_patch_name}"
 done
-sudo cp "$current" "$target"
+
+if [[ "$current" != "$target" ]]; then
+    sudo cp "$current" "$target"
+fi
+}
+
+function patch_node_ignition() {
+    local kind
+    local bootstrap
+    local wd
+    local config
+
+    kind="$1"
+    bootstrap="$2"
+    wd="$(mktemp -d)"
```

Comment:
we appear to be leaking temp directories, but we can fix this up in another patch as there is another one to be fixed that isn't part of this patch.